### PR TITLE
docs: add a PR template and a Contributing.md file

### PR DESCRIPTION
**Pull request by joanise**
### PR Goal? 

To add a standard template guiding PR submitters to provide better comments for reviewers.

Also, noticed that there was no Contributing.md. I added one, but I think it needs more work...

### Fixes? 

n/a

### Feedback sought? 

If you're happy with the template, which I'm using here, and suggestions for fleshing out Contributing.md

### Priority? 

low

### Tests added? 

n/a

### How to test? 

You can try it out with this link: https://github.com/joanise/just-testing/compare/main...dev.gzip-on-ci-all-OSs?expand=1&template=studio_web_pr_template.md

### Confidence? 

The template works fine in EveryVoice already, so I assume it'll be good here, but low confidence in Contributing.md

### Version change? 

no

### CODE REVIEW

File: Contributing.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+These rules are inspired by these commit formatting guides:
+
+- [Conventional Commits](https://www.conventionalcommits.org/)
+- [Bluejava commit guide](https://github.com/bluejava/git-commit-guide)
+- [develar's commit message format](https://gist.github.com/develar/273e2eb938792cf5f86451fbac2bcd51)
+- [AngularJS Git Commit Message Conventions](https://docs.google.com/document/d/1QrDFcIiPjSLDn3EL15IJygNPiHORgU1_OOAqWjiDU5Y).
+
+With commitlint enabled, your commit log messages will be checked:
+
+- `git commit -m'fixing a bug in g2p integration'` outputs an error
```
Comment (roedoejet) [inline Contributing.md:73]:
```suggestion
- `git commit -m 'fixing a bug in g2p integration'` outputs an error
```

File: Contributing.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+- `git commit -m'fix(g2p): fixing a bug in g2p integration'` works
```
Comment (roedoejet) [inline Contributing.md:74]:
```suggestion
- `git commit -m 'fix(g2p): fixing a bug in g2p integration'` works
```

File: Contributing.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
+# Contributing to ReadAlong Studio-Web
+
+Feel free to dive in! [Open an issue](https://github.com/ReadAlongs/Studio-Web/issues/new) or submit PRs.
+
+This repo follows the [Contributor Covenant](http://contributor-covenant.org/version/1/3/0/) Code of Conduct.
+
+This repo uses automated tools to standardize the formatting of code, text files and commits.
+
+- The [pre-commit hook](#pre-commit-hook) automatically applies code formatting rules using `prettier`.
+- [commitlint](#commitlint) is used as a commit message hook to validate that commit messages follow the convention.
```
Comment (roedoejet) [inline Contributing.md:10]: Looks good to me - I might make mention that this is a mono-repo managed by nx though. Stating that dependencies should be defined at the top level.

Comment (roedoejet) [reply Contributing.md:10]: and for information about spinning up development environments, consult the main readme.

**Review by roedoejet — APPROVED** [review]
A few small suggestions - approving in advance though. Thanks @joanise

File: Contributing.md (hunk 1 above)
Comment (joanise) [reply Contributing.md:73]: lol

File: Contributing.md (hunk 2 above)
Comment (joanise) [reply Contributing.md:74]: Makes me think of the Silicon Valley episode where he dates someone who prefers spaces over tabs... 😆

**Review by deltork — APPROVED** [review]
Looks good to me

File: Contributing.md (hunk 3 above)
Comment (joanise) [reply Contributing.md:10]: done

**Pull request merged**